cmd/gitops/add/app/runner.go

```diff
@@ -68,6 +68,14 @@ func (r *runner) run(ctx context.Context, cmd *cobra.Command, args []string) err
 		config.AppName = config.App
 	}
 
+	if cmd.Flags().Changed("inCluster") && r.flag.InCluster {
```

Comment:
Only add it when the flag is explicitly passed. This ensure the behavior has not change by default